pkg/core/consensus/candidate/blockgenerator.go

```diff
@@ -103,7 +103,13 @@ func (sf ScoreFactory) Create(sender []byte, round uint64, step uint8) consensus
 func (bg *Generator) Collect(e consensus.InternalPacket) error {
 	sev := e.(message.ScoreProposal)
 
-	blk, err := bg.Generate(sev)
+	resp, err := bg.rpcBus.Call(topics.GetLastCommittee, rpbus.EmptyRequest(), 5*time.Second)
+	if err != nil {
+		return err
+	}
+	keys := resp.([][]byte)
```

Comment:
This will throw an error. The Chain responds to `GetLastCommittee` requests with a `bytes.Buffer` carrying the wire-encoded BLS keys of the committee (which I think it is unnecessary). Either change `chain.provideLastCommittee` to return a `[][]byte` or change this line to cast to a `bytes.Buffer`

Reply:
My preference goes for the former of the 2 options.